js/index.js

```diff
@@ -359,6 +359,7 @@ function renderFrame() {
 
   if (!PLAYER.paused && !PLAYER.muted) {
     requestAnimationFrame(renderFrame)
+    stopTimestamp = Date.now()
```

Comment:
It shouldn't be set here on every iteration. Put it in 'pause' and 'volumechange' player's event handlers